Backup/simple-backup-service-faqs.md

```diff
@@ -139,7 +139,7 @@ A: Yes, in the restore section there is an option to perform a full restore, or
 
 **Q: Can restores be performed to another server?**
 
-A: A self-service feature is on the SBS roadmap. If a restore to an alternate server is required, please create a [support request](https://www.ctl.io/knowledge-base/support/how-do-i-report-a-support-issue/#exceptions).
+A: Yes, please follow the steps outlined in this [KB article](https://www.ctl.io/knowledge-base/backup/restore-to-alternate-server/).
```

Comment:
@Ian-Davidson Please use the relative link to the .md file.
since it is in the same directory as this article, you can use `restore-to-alternate-server.md`

Reply:
@Ian-Davidson I took care of it for you.